add pretrained model description and saving disk description (#5383)
opensearch-project/documentation-website: _search-plugins/neural-sparse-search.md

```
{% include copy-curl.html %}

> Note here you can exclude embedding vector source. It will save tons of disk space. Do it like:

Reviewer: ```suggestion
To save disk space, you can exclude the embedding vector source as follows:

```

```json
PUT /my-nlp-index
{
  "settings": {
    "default_pipeline": "nlp-ingest-pipeline-sparse"
  },
  "mappings": {
      "_source": {
      "excludes": [
        "passage_embedding"
      ]
    },
    "properties": {
      "id": {
        "type": "text"
      },
      "passage_embedding": {
        "type": "rank_features"
      },
      "passage_text": {
        "type": "text"
      }
    }
  }
}

> ```

Reviewer: ````suggestion
```
{% include copy-curl.html %}

````

> Once the `<token, weights>` terms have been excluded from the source, they cannot be recovered. It is a trade-off so please make sure you don't these  `<token, weights>` pairs before you applied this optimization.

Reviewer: ```suggestion
Once the `<token, weight>` pairs are excluded from the source, they cannot be recovered. Before applying this optimization, make sure you don't need the  `<token, weight>` pairs for your application.
{: .important}
```
